zambeze/orchestration/db/model/abstract_entity.py

```diff
@@ -0,0 +1,32 @@
+from abc import ABCMeta, abstractmethod
+
+
+class AbstractEntity(object, metaclass=ABCMeta):
+    @classmethod
+    @property
+    @abstractmethod
+    def ID_FIELD_NAME(cls) -> str:
+        raise NotImplementedError()
+
+    @classmethod
+    @property
+    @abstractmethod
+    def FIELD_NAMES(cls) -> str:
+        raise NotImplementedError()
+
+    @classmethod
+    @property
+    @abstractmethod
+    def ENTITY_NAME(cls) -> str:
+        raise NotImplementedError()
```

Comment:
Is this what you meant about constants? This is essentially a constant member of the class right?

Reply:
We should add something in the CONTRIBUTING.md page about what is expected for constants.

Reply:
> What does passing in (cls) as the parameter do?

This is the "self" argument but for a class-level method. Since this is supposed to be a class-level (as opposed to instance-level) constant, I believe `cls` is right.